### ISSUE TITLE
Installation ubuntu 18.04 - with workaround :D

### ISSUE THREAD

**Issue by aluvare** [kimchi-project/kimchi#1242]
The library python-imaging was changed in ubuntu 18.04 to python-pil.

Kimchi dependencies in ubuntu 18.04:
```bash
apt install python-pil python-configobj novnc python-libvirt libvirt-bin nfs-common qemu-kvm python-parted python-ethtool python-ipaddr python-guestfs libguestfs-tools spice-html5 python-magic python-paramiko
```
Patch:
```bash
ar x kimchi-2.5.0-0.noarch.deb
tar -zxvf control.tar.gz
sed -i 's/python-imaging/python-pil/g' control
tar c Makefile Makefile.am Makefile.in control control.in | gzip -c > control.tar.gz
ar rcs kimchi-2.5.0-0.noarch_patched.deb debian-binary control.tar.gz data.tar.xz
```
And after that:
```bash 
dpkg -i kimchi-2.5.0-0.noarch_patched.deb
```

**Comment by hotnikq** [comment]
kimchi : Depende: wok (>= 2.1.0) mas não é instalável
          Depende: python-jsonschema (>= 1.3.0) mas não será instalado
          Depende: python-psutil (>= 0.6.0) mas não será instalado
          Depende: sosreport mas não será instalado
          Depende: python-lxml mas não será instalado

**Comment by hotnikq** [comment]
O pacote 'wok' não tem candidato para instalação

**Comment by hotnikq** [comment]
I will clear install ubuntu and try again soon,
my kvm server have gpu passthrough
with nvidia Nvs card for host with Nouveau blacklist "hard to do"
it will take some weeks.

**Comment by RootG** [comment]
With this [pull request](https://github.com/kimchi-project/kimchi/pull/1260) patch to this issue has been merged. However, it is not released yet.

See [maintainer's answer](https://github.com/kimchi-project/kimchi/issues/1246#issuecomment-500575117).

**Comment by alinefm** [comment]
It is now fixed on Kimchi 3.0: https://github.com/kimchi-project/kimchi/releases/tag/3.0.0
